.github/workflows/test.yml

```
@@ -35,17 +35,21 @@ jobs:
     - run: npm test
   tox:
     name: >-
-      ${{ matrix.env.TOXENV }}
+      ${{ matrix.tox_env }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        python-version:
-        - 3.8
-        os:
-        - ubuntu-20.04
-        env:
-        - TOXENV: py,lint
+        include:
+        - python-version: 3.8
+          os: ubuntu-20.04
+          tox_env: py38,lint
+        - python-version: 3.9
+          os: ubuntu-20.04
+          tox_env: py39,lint
+        - python-version: '3.10'
+          os: ubuntu-20.04
+          tox_env: py310,lint
     env:
       TOX_PARALLEL_NO_SPINNER: 1
```

Instead of passing an explicit CLI arg in each invocation, it's common to just put it into an env var like this:
```suggestion
      TOXENV: ${{ matrix.toxenv }}
      TOX_PARALLEL_NO_SPINNER: 1
```

---

done as suggested